packages/blade/src/components/Tooltip/TooltipContent.tsx

```diff
@@ -21,16 +21,15 @@ const TooltipContent = React.forwardRef<HTMLDivElement, TooltipContentProps>(
         isVisible={isVisible}
       >
         {title ? (
-          <Text contrast="high" weight="semibold" size="medium">
+          <Text weight="semibold" size="medium" color="surface.text.staticWhite.normal">
             {title}
           </Text>
         ) : null}
         <Text
           variant="body"
           size="small"
           weight="regular"
-          contrast="high"
-          color="feedback.text.neutral.highContrast"
+          color="interactive.text.staticWhite.subtle"
```

Comment:
this shouldn't be interactive right? it shall be surface?

Reply:
updated.